com.unity.robotics.urdf-importer/Runtime/RuntimeImport/RuntimeURDF.cs

```diff
@@ -0,0 +1,214 @@
+using System;
+using UnityEngine;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+/// <summary>
+/// Wrappers for Editor calls used in the URDF Importer functions.
+/// This is to reduce the usage of #if UNITY_EDITOR in the code and have a consistent way of skipping editor only code. 
+/// Also to allow having the option of running the runtime mode code in editor.
+/// </summary>
+public static class RuntimeURDF
+{
+#if UNITY_EDITOR
+    public static bool runtimeModeEnabled = false;
+#else
+    public static bool runtimeModeEnabled = true;
+#endif
+    public static bool IsRuntimeMode() 
+    {
+        return runtimeModeEnabled;
+    }
+
+    public static void SetRuntimeMode(bool enabled)
+    {
+        runtimeModeEnabled = enabled;
+    }
+
+    public static T AssetDatabase_LoadAssetAtPath<T>(string fileAssetPath) where T : UnityEngine.Object 
```

Comment:
I am not sure whether the naming of the methods obeys the coding best practice, but I cannot think of a better idea to name them. @mrpropellers

Reply:
I agree. I wasn't too happy either, but I wanted to make it clear what the actual functions the wrappers present. We can easily rename them all later in a separate PR as they are all declared in a single file.